go.mod

```diff
@@ -4,7 +4,6 @@ go 1.12
 
 require (
 	cloud.google.com/go v0.39.0 // indirect
-	github.com/DataDog/dd-trace-go v1.14.0
```

Comment:
`go.mod` changes should not be needed here?

Reply:
@miekg Ah it actually points to `gopkg.in/DataDog/dd-trace-go.v1 v1.14.0` now. See  Line 70 below.

Reply:
but do we want all the other updates here as well?

Reply:
I think in PR #2868, three copies were added (v0.6.1 was the old one):

```
        github.com/DataDog/dd-trace-go v1.14.0
	gopkg.in/DataDog/dd-trace-go.v0 v0.6.1		
 	gopkg.in/DataDog/dd-trace-go.v1 v1.14.0
```

After `go mod tidy` in this PR, two are removed:
```diff
-       github.com/DataDog/dd-trace-go v1.14.0
-	gopkg.in/DataDog/dd-trace-go.v0 v0.6.1		
 	gopkg.in/DataDog/dd-trace-go.v1 v1.14.0
```

Reply:
@miekg Do you mean you want to update `go mod` to bring everything update to date (minus client-go as it has issues with compatibility)?